src/js/components/dashboard/DashboardTable.jsx

```diff
@@ -164,34 +176,39 @@ export default class DashboardTable extends React.Component {
             }
 
             // break the object out into an array for the table component
-            const row = 
-            [
-                <SubmissionLink submissionId={item.submission_id} />,
-                item.agency,
-                reportingDateString,
-                userName,
-                this.convertToLocalDate(item.last_modified),
-                <Status.SubmissionStatus status={item.rowStatus} certified={this.props.isCertified} />
-            ];
-
-            if(!this.props.isCertified && PermissionsHelper.checkAgencyPermissions(this.props.session)) {
-                if(item.publish_status === "unpublished" && PermissionsHelper.checkAgencyPermissions(this.props.session, item.agency)) {
-                    row.push(<DeleteLink submissionId={item.submission_id} index={index} warning={this.deleteWarning.bind(this)} confirm={deleteConfirm} reload={this.reload.bind(this)} item={item} account={this.state.account}/>);
-                }
-                else {
-                    row.push("N/A");
-                }   
-            }
-            else if(this.props.isCertified) {
-                row.push(item.certifying_user);
-
-                // get certified on date and process it
-                let certified_on = item.certified_on;
-                if(certified_on !== "") {
-                    // call a function to convert/format the date properly
-                    certified_on = this.convertToLocalDate(certified_on);
+            let row = [];
```

Comment:
Arrays should be `const` because while the value within the array is changing, the variable is still that same array